tests/test_main.py

```diff
@@ -1,8 +1,15 @@
 # -*- coding: utf-8 -*-
 import unittest
 
-from language_detector import detect_language, LANGUAGES
+#import detect_language
+#from language_detector import LANGUAGES
+import sys 
+
+sys.path.append('/home/ubuntu/workspace/language_detector')
+sys.path.append('/home/ubuntu/workspace/')
+#print ("\n".join(sys.path))
```

Comment:
I'm curious as to why you decided to change the test case by adding these sys lines, I guess you had problems importing your .py files while executing Travis? Seems like you added a utf-8 comment and changed the setup.py document as well, maybe that's why you're getting problems with Travis?